craco.config.cjs

```diff
@@ -7,15 +7,30 @@ const commitHash = require('child_process').execSync('git rev-parse HEAD')
 
 module.exports = {
   babel: {
-    plugins: ['@vanilla-extract/babel-plugin'],
-    env: {
-      test: {
-        plugins: ['istanbul'],
-      },
-      development: {
-        plugins: ['istanbul'],
-      },
-    },
+    plugins: [
+      '@vanilla-extract/babel-plugin',
+      ...(process.env.REACT_APP_UNSAFE_EVAL
```

Comment:
seems like you're mixing concerns here with using this env variable. could you set something like ENV=E2E_TEST or something like that?